src/test/java/com/solvd/carina_testing/HomePageTest.java

```diff
@@ -0,0 +1,78 @@
+package com.solvd.carina_testing;
+
+import com.solvd.carina_testing.components.*;
+import com.solvd.carina_testing.pages.CheckoutPage;
+import com.solvd.carina_testing.pages.HomePage;
+import com.solvd.carina_testing.pages.SearchPage;
+import com.zebrunner.carina.core.AbstractTest;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+public class HomePageTest extends AbstractTest {
+    @Test(testName = "verify search line test")
```

Comment:
add empty line between className and test
If you decided to add testName, capitalize the first letter